gym/envs/registration.py

```diff
@@ -7,34 +7,29 @@
 import importlib.util
 import re
 import sys
+from dataclasses import dataclass, field
 from typing import (
     Any,
     Callable,
-    Generator,
     Optional,
     Sequence,
     SupportsFloat,
     Tuple,
     Type,
     Union,
-    cast,
     overload,
 )
 
+import numpy as np
+
+from gym.envs.__relocated__ import internal_env_relocation_map
+from gym.wrappers import AutoResetWrapper, OrderEnforcing, TimeLimit
+
 if sys.version_info < (3, 10):
     import importlib_metadata as metadata  # type: ignore
 else:
     import importlib.metadata as metadata
 
-from collections import defaultdict
-from collections.abc import MutableMapping
-from dataclasses import InitVar, dataclass, field
-
-import numpy as np
-
-from gym import Env, error, logger
-from gym.envs.__relocated__ import internal_env_relocation_map
-
 if sys.version_info >= (3, 8):
```

Comment:
As we build for python 3.7 and greater then this is not needed anymore

Reply:
I tested for python 3.7 and without the `__future__` then the import fails however with the `__future__` I didnt have an issue